lldb/packages/Python/lldbsuite/test/lldbtest.py

```diff
@@ -1572,6 +1572,15 @@ def findBuiltClang(self):
 
         return os.environ["CC"]
 
+    def getBuiltinServerTool(self, server_tool):
+        # Tries to find simulation/lldb-server/gdbserver tool at the same folder as the lldb.
+        lldb_dir = os.path.dirname(lldbtest_config.lldbExec)
+        path = shutil.which(server_tool, path=lldb_dir)
+        if path is not None:
+            return path
+
+        return ""
```

Comment:
return None instead of an empty string

Reply:
Agreed with this comment.
I would like to modify this function to get the lldb-server/gdbserver based on OS.
Is this fine ?